www/experiments/lcp.inc

```diff
@@ -26,13 +26,8 @@
         
             if (isset($lcp['type']) && $lcp['type'] === "image") {
                 if (isset($lcp['element']['src']) || isset($lcp['element']['currentSrc'])) {
-                    $lcpSource = $lcp['element']['src'];
-                    $lcpFullSource = $lcp['element']['src'];
-                    // get actual source attribute if we can.
-                    $srcAttrVal = preg_match_all('/src\=\"([^"]+)/i', $lcp['element']['outerHTML'], $srcmatches );
```

Comment:
@scottjehl I killed the srcAttrVal bit as I don't think we need that, but not sure if there was an edge case you ran into that I'm not aware of?

Reply:
Ah...ok, that makes sense. I'll bring this back then.